### ISSUE TITLE
.conf not properly formatted as of 3.4.3

### ISSUE THREAD

**Issue by jasonwilliams14** [nginxinc/kubernetes-ingress#5184]
**Describe the bug**
The underlying nginx.conf is not being properly generated. Directives that should be on their own line are not which makes reading the .conf difficult.

**To Reproduce**
Install 3.4.3, add several resources and review the generated .conf

Latest results as of today using 3.4.3 (using prebuilt image and built my own)

```
upstream vs_cafe-test {
    zone vs_cafe_test 512k;random two least_conn;
```

In the above, `random two least_conn` should and needs to be on its own line.

**Expected behavior**
See above.

**Your environment**

Tested with Kind and K3D, version 3.4.3 of NIC.

**Comment by j1m-ryan** [comment]
This is [fixed](https://github.com/nginxinc/kubernetes-ingress/pull/4958/files) on `main` but wasn't cherry picked into `v3.4.3`

**Comment by vepatel** [comment]
@jasonwilliams14 duplicate of https://github.com/nginxinc/kubernetes-ingress/issues/4951

**Comment by jjngx** [comment]
fix for this will land in release v3.5.0
